dist/src/main/java/io/camunda/zeebe/gateway/GatewayConfiguration.java

```diff
@@ -42,6 +43,11 @@ public Duration shutdownTimeout() {
     return lifecycleProperties.getTimeoutPerShutdownPhase();
   }
 
+  @Bean
+  public BrokerClientTimeoutConfiguration brokerClientConfig() {
+    return new BrokerClientTimeoutConfiguration(config.getCluster().getRequestTimeout());
+  }
```

Comment:
Can this also done once?

Reply:
If you mean by that if the `BrokerClientTimeoutConfiguration` can be declared only once, then no. It depends, the request timeout is retrieved differently depending, if the broker or the (standalone) gateway is started. Or in other words, as long as we have different configurations for the broker (with embedded gateway) and the gateway itself, that bean must declared twice but depending on the profile it will registered and initiated only once.